src/docs/tutorials/04-edit-email-templates-with-flex-cli/index.md

```diff
@@ -188,19 +188,88 @@ text editor and make a simple change:
 
 When you've made the change, save the file.
 
+## Preview your changes
+
+You can test your changes and templates by previewing or sending a test
+email using your local template files. To preview the change above:
+
+```bash
+flex-cli notifications preview -m my-test-marketplace --template preauth-nightly-booking/templates/new-booking-request
+```
+
+The command will render the given template and open a browser tab with
```

Comment:
"The command will render the given template using a sample data (context) and open a browser to preview it".
Here we could also link to the sample data context.